src/Bounds.cpp

```diff
@@ -1664,7 +1700,7 @@ Region region_union(const Region &a, const Region &b) {
         Expr max_plus_one = Max::make(max_a, max_b);
         Expr extent = max_plus_one - min;
         result.push_back(Range(simplify(min), simplify(extent)));
-        //result.push_back(Range(min, extent));
+        // result.push_back(Range(min, extent));
```

Comment:
Is there any reason to not remove this line?

Reply:
I didn't touch this, clang-format just tweaked the comment. But yeah, I see no reason for it to stay.